Library/Formula/clang-format.rb

```diff
@@ -1,18 +1,18 @@
 class ClangFormat < Formula
   desc "Formatting tools for C, C++, Obj-C, Java, JavaScript, TypeScript"
   homepage "http://clang.llvm.org/docs/ClangFormat.html"
-  version "2015-09-01"
+  version "3.7.1"
```

Comment:
Yeah unfortunately we can’t do really much for now. This is an issue for a few formula and we discussed about a fix in the past but there’s nothing for now.